dces-drc-integration/src/main/resources/application.yaml

```diff
@@ -104,6 +104,8 @@ springdoc:
     path: /swagger-ui-custom.html
     enabled: true
     operations-sorter: method
+    api-url: https://${API_URL}
+    oauth-token-url: ${DCES_DRC_API_OAUTH_TOKEN_URL}
```

Comment:
Is this coming from `config-variables`? Can it have a default value that fails gracefully (e.g. by just not authenticating the try now) if it's not set?

Or if try now not working, then should it be removed?

Reply:
It does come from `config-variables`. I left it in because it does successfully authorise with valid client ID and secret, so it's one step in the right direction. I'll set a default value that says that the variable is not set, for clarity.